.github/ISSUE_TEMPLATE/performance_issue_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Performance issue report
+about: Scalability is our top priority
```

Comment:
Think we need to add a bit more useful info here, something along the lines of, are you experiencing slow load/response times, high resource utilization, etc?